### PR TITLE
Task # 38 - Splits introductory vignette into 2.

### PR DESCRIPTION
**Pull request by ccamara**
As per https://github.com/ITSLeeds/pct/issues/38 request.

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Looks good to me @ccamara, I resolved some conflicts (did you start with the most recent, e.g. recently cloned) version of the repo? If it passes the CI tests above I plan to merge it and add you to as a contributor, if you like (do you have an ORCID ID if so?).

File: vignettes/reproducing_pct_results.Rmd [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+line_order = order(wight_lines_pct$bicycle, decreasing = TRUE)
+wight_lines_30 = wight_lines_pct[line_order[1:30], ]
+```
+
+The resulting `wight_lines_pct` and `wight_lines_30` datasets are available in the package.
+We'll use the smaller one for speed.
+Note: these contain many variables, three of which (the number of people cycling, driving and walking along the desire lines from the 2011 Census) are shown below for the Isle of Wight:
+
+```{r, pct-lines-min}
+lwd = wight_lines_30$all / mean(wight_lines_30$all) * 5
```
Comment (Robinlovelace) [inline vignettes/reproducing_pct_results.Rmd:51]: The problem here is that the object `wight_lines_30` has not yet been created. I suggest starting from a more recent version of the package (I made a load of changes to the `pct.Rmd` file in the last few days) may be the best way forward.

**Review by Robinlovelace — CHANGES_REQUESTED** [review]
Looks good but tests are not passing for reasons outlined in a comment. Happy to talk over how to best proceed but it may be best starting a new PR based on the latest version of the repo.

**Comment by ccamara** [comment]
Ok, will do it this weekend (as you can see, I'm just doing that on weekends and you may have updated the repo since last time I forked it) with the same date version (probably in another PR, as I don't want to lose time fixing conflicts)!

**Comment by Robinlovelace** [comment]
Thanks @ccamara if you have any questions on updating it just let me know.

**Pull request closed without merging**